.github/ISSUE_TEMPLATE/3-performance-issue-report.yml

```diff
@@ -0,0 +1,68 @@
+name: 🔧 Performance issue report
+description: Report a performance issue if something isn't performing well in WooCommerce Core.
+title: "[Performance]: "
+labels: [ "type: enhancement", "focus: performance"]
```

Comment:
Default issue labels.

Reply:
If these are going to a Flux project board for further triaging could we maybe add a Flux label here so they don't show up in the main triaging (of issues without team labels) where they likely would be given a Flux label initially anyway?